test-integration/scores.coffee

```diff
@@ -0,0 +1,127 @@
+{describe, CourseSelect, screenshot} = require './helpers'
+selenium = require 'selenium-webdriver'
+{expect} = require 'chai'
+
+TEACHER_USERNAME = 'teacher01'
+
+MAIN_ELEMENTS = [
+  '.scores-report',
+  '.course-scores-wrap',
+  '.nav-tabs',
+  '.export-button',
+  '.course-scores-title',
+  '.course-scores-container'
+]
+
+
+class StudentScores
+
+  constructor: (@test, @login, @courseTitle) ->
```

Comment:
Because this helper class is intended to test only the student dashboard. In order to open the student dashboard you need a login and a course, and I didn't want to have to include the three/four lines to navigate to it the spec.  If they were in the spec, it's possible they could be missed or buggy and then `StudentDashboard` wouldn't work properly.

I think the scenario you outlined would be better tested by using other helper classes.  The course chooser already exists and is used internally by the `StudentDashboard` class.